Doc/library/asyncio-stream.rst

```diff
@@ -42,6 +41,18 @@ The following top-level asyncio functions can be used to create
 and work with streams:
 
 
+.. coroutinefunction:: connect(host=None, port=None, \*, \
+                               loop=None, limit=None, ssl=None, family=0, \
+                               proto=0, flags=0, sock=None, local_addr=None, \
+                               server_hostname=None, ssl_handshake_timeout=None, \
+                               happy_eyeballs_delay=None, interleave=None)
+
+   This function internally uses :meth:`loop.create_connection` to return a
+   :class:`Stream` object of the mode ``READWRITE`` that can be used as a reader
+   and a writer.
```

Comment:
Please reflect the fact that `async with connect()`  can be used as well as just `await connect()`.

Reply:
I have added a note that connect returns a stream object which is awaitable and also about using it as context manager. Let me know if it needs to be reworded that connect itself is awaitable.